lib/rufus/lua/state.rb

```diff
@@ -244,8 +241,7 @@ def stack_push_array(a)
     # Loads a Lua global value on top of the stack
     #
     def stack_load_global(name)
-
-      Lib.lua_getfield(@pointer, LUA_GLOBALSINDEX, name)
+      Lib.lua_getglobal(@pointer, name)
```

Comment:
Since the Global pseudo index was removed in Lua 5.2, we're using `lua_getglobal` method instead (for 5.3):

https://www.lua.org/source/5.3/lapi.c.html#lua_getglobal

This is shorthand for `getfield` with the Global index:
https://www.lua.org/source/5.3/lapi.c.html#lua_getfield